_posts/2022-03-16-launch-announcement-1-3-0.markdown

```diff
@@ -0,0 +1,109 @@
+
+![Field Insights View]({{ site.baseurl }}/assets/media/blog-images/2022-03-16-Launch-Announcement-1-3-0/field-insights.gif){: .img-fluid }
+
+### SQL and PPL
+
+* **PPL Runtime Fields**: Users may opt out of formatting the schema upon writing the index (schema-on-write). This opt out can be used to speed up indexing time, or for additional presentation flexibility. Users who want to define their schema upon querying the index (schema-on-read), can now use regex and parse commands to further format their schema using PPL.
+
+![PPL runtime workflow]({{ site.baseurl }}/assets/media/blog-images/2022-03-16-Launch-Announcement-1-3-0/PPL-Parse.gif){: .img-fluid }
+
+
+* **Machine Learning Support in PPL (Piped Processing Language):** Users can now process arbitrary observability events using Anomaly Detection based on Random Cut Forest (RCF) and KMeans commands in PPL.
+* **Improved Aggregate Functions Support in SQL and PPL**: Users expect to use aggregate functions in SQL and PPL. OpenSearch SQL and PPL now supports ORDER BY and default query limit size.
+* **Support for Group Field and Span in Stats Command**: Users can now use group fields together and can span by a specified interval using the Stats command in PPL. Also, users can now create multiple time series’ with a single command.
+* **Comma-Separated Index Matching**: Users can now query multiple indices using a comma separated value using PPL.
+* **CAST Function Supported in PPL**: Users can now change datatypes using the CAST function in PPL.
+* **Support IN Clause in SQL and PPL**: Users can now use the IN clause within SQL and PPL to select from within a value list.
+* **Support date_nanos in SQL and PPL**: Users can now search on indices with the field type of data_nanos.
+
+### Alerting
+
+* **Cluster Metric Monitors**: Users can now create monitors within Alerts which help administrators feel more confident about managing the health of their clusters. The new monitor reports on dimensions including, but not limited to CPU usage, JVM memory usage, and total number of documents coming into the cluster.
+
+### Anomaly Detection
+
+* **New Detector Validation**: When setting up a new anomaly detector, the parameters specified will be validated to ensure that the criteria can initialize and that the detector will perform. The validation presented in the user interface and API will prevent detectors from being created which will not initialize.
+
+### ML Commons
+
+* **Machine Learning (ML) Commons:** A new solution that makes it easy to develop new machine learning features. It allows engineers to leverage existing open source machine learning algorithms and reduce the efforts to build them from scratch. It also removes the necessity from engineers to manage the machine learning tasks which will help to speed the feature developing process.
+* **K-Means and Random Cut Forest Algorithm Support**:
```

Comment:
In other part of the document it's written as 'Kmeans'

Reply:
Let's keep consistent, seems "K-means" is mostly used

Reply:
Got it!